monai/handlers/checkpoint_saver.py

```diff
@@ -62,6 +62,8 @@ class CheckpointSaver:
             typically, it's used to resume training and compare current metric with previous N values.
         key_metric_greater_or_equal: if `True`, the latest equally scored model is stored. Otherwise,
             save the the first equally scored model. default to `False`.
+        key_metric_score_sign: sign of the score: `1.0` or `-1.0`. For error-like metrics, e.g. smaller is better,
```

Comment:
This could be an int or Boolean flag

Reply:
Thanks for your review, updated doc-string in the latest commit.